### PR TITLE
when direction of render is changed, add the margin to the icons

### PR DESCRIPTION
**Pull request by sheela-svmx**
When the browser direction is Right to Left or  Left to Right we need to override the margin-right which is present

Issue: When the user of language Hebrew or arabic is viewed the direction of render would be "rtl" and the margins on the icons were making the render on "rtl" wonky"

Fix: To use -webkit-margin-start/ -webkit-margin-end which will , If the writing direction is left-to-right, this value overrides margin-left. If the writing direction is right-to-left, this value overrides margin-right.
Ref jira :https://servicemax.atlassian.net/browse/UI-3074

### CODE REVIEW

**Comment by sheela-svmx** [comment]
@joshsylvester @mallik-svmx Help review

File: bower.json [hunk 1]
```diff
@@ -1,9 +1,7 @@
 {
   "name": "px-datetime-common",
-  "version": "3.2.4",
-  "main": [
-    "px-datetime-common.html"
-  ],
+  "version": "3.2.5",
+  "main": ["px-datetime-common.html"],
```
Comment (joshsylvester) [inline bower.json]: let's revert the formatting, please.

Comment (sheela-svmx) [reply bower.json]: Sure yes updated

File: HISTORY.md [hunk 2]
```diff
@@ -1,349 +1,353 @@
-v3.2.4
```
Comment (mallik-svmx) [inline HISTORY.md]: remove formatting diff.

Comment (sheela-svmx) [reply HISTORY.md]: I need to turn off my prettier, it is reformatting, will change

**Comment by arongraham** [comment]
nice work @mallik-svmx and @joshsylvester! 

thanks for contributing, we really appreciate these updates to the design system and its components

**Pull request merged**